modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} router for dual-stack support
+
+When you install a {product-title} cluster on bare metal infrastructure, you can configure the router for single-stack or dual-stack support.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+////
+. Open the Network YAML file by entering the following command:
++
+[source,terminal]
+----
+$ oc get network/cluster -o yaml
+----
+////
+
+. To configure the router for single-stack or dual-stack support, create a service:
++
+[source,terminal]
+----
+$ oc get service <service_name>
+----
++
+. Check your services by entering the following command:
++
+[source,terminal]
+----
+$ oc get service <service_name> -o yaml
+----
++
+. Edit the `ipFamilies` and `ipFamilyPolicy` fields in the service YAML file:
++
+.Sample Network YAML file
+[source,yaml]
+----
+apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: yyyy-mm-ddT00:00:00Z
+    labels:
+      name: <service_name>
+    managedFields:
+    - apiVersion: v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        f:metadata:
+          f:labels: {}
+            .: {}
+            f:name: {}
+        f:spec: {}
+          f:ipFamilies: {}
```

Comment:
Probably don't need to include the `managedFields` data, and a good about of the object metadata.